bin/write_probspecs_info.nim

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env -S nim r --verbosity:0 --skipParentCfg:on
+import std/[algorithm, json, os, osproc, strformat, strutils, times]
+import pkg/jsony
+
```

Comment:
I put the script in `bin/`, rather than `src/info` - I think in `bin/` it's a little clearer that the file is a script for generating a JSON file, rather than a `.nim` file that's compiled when calling `nimble build`.

But I'm OK with `src/info` if you prefer.

Reply:
No, `bin/` is fine, as we're using that in many repos.